frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -48,22 +54,65 @@ export const surveysLogic = kea<surveysLogicType>([
         setSearchTerm: (searchTerm: string) => ({ searchTerm }),
         setSurveysFilters: (filters: Partial<SurveysFilters>, replace?: boolean) => ({ filters, replace }),
         setTab: (tab: SurveysTabs) => ({ tab }),
+        loadNextPage: true,
+        loadBackendSearchResults: true,
     }),
     loaders(({ values }) => ({
         surveys: {
-            __default: [] as Survey[],
+            __default: { results: [], count: 0 } as CountedPaginatedResponse<Survey>,
             loadSurveys: async () => {
-                const responseSurveys = await api.surveys.list()
-                return responseSurveys.results
+                return await api.surveys.list()
+            },
+            loadNextPage: async () => {
+                if (!values.surveys.next || !isURL(values.surveys.next)) {
+                    return values.surveys
+                }
+
+                const url = new URL(values.surveys.next)
+                const limit = parseInt(url.searchParams.get('limit') || SURVEY_PAGE_SIZE.toString())
```

Comment:
if we just use the `count` as suggested [here](https://github.com/PostHog/posthog/pull/29552/files#r1983220032), we can just avoid this URL parsing etc

Reply:
Btw i recall browsers (firefox/chrome) having some issues with URL specifically, but I guess `isURL` with the regex would do the job before calling `URL(x)`

Reply:
okay that makes sense yeah, i'll use `count` instead, will update in a bit